pocs/tests/test_database.py

```diff
@@ -34,8 +34,29 @@ def test_insert_and_no_collection(db):
     db.current.remove({'type': 'config'})
 
 
+def test_simple_insert(db):
+    rec = {'test': 'insert'}
+    db.insert('config', rec)
+
+    record = db.config.find({'data.test': {'$exists': 1}})
```

Comment:
This seems very mongo specific. Are you not attempting to address that ... yet?

Reply:
Correct. I was thinking of actually renaming to `test_mongo.py`, which could be part of this PR. Actually addressing anything non-mongo is for another PR.

Reply:
Renamed to `test_panmongo.py` for now.